cli/src/cli_util.rs

```diff
@@ -1006,6 +1006,14 @@ impl WorkspaceCommandHelper {
         ui: &mut Ui,
     ) -> Result<Commit, CommandError> {
         let revset_expression = self.parse_revset(revision_str, Some(ui))?;
+        let branch: Option<String> =
+            if let RevsetExpression::CommitRef(RevsetCommitRef::Symbol(ref branch_name)) =
```

Comment:
Is the `ref` necessary?

Reply:
I added the `ref` because otherwise `RevsetExpression` would be moved out from `revset_expression`

Reply:
I see. I think you can do `revset_expression.as_ref()` instead. Does that work?